lib/sqlalchemy/cyextension/collections.pyx

```diff
@@ -147,7 +151,7 @@ cdef class OrderedSet(set):
         self.intersection_update(other)
         return self
 
-    def symmetric_difference_update(self, other):
+    cpdef symmetric_difference_update(self, other):
```

Comment:
if we decide on using cpdef here, it may make sense to use it also in the other methods called by `__<op>__`, like difference, etc

Reply:
The other methods has starred argument, so they cannot be directly changed to `cpdef`. Of course, one can create `cdef` method and call it from `def` method but I wanted to keep changes light. I can implemented that if you are OK with it.

Reply:
Interesting, I did not know you couldn't use it with args methods.

Nevermind then. Thanks for taking a look